src/components/Facility/BedCapacity.tsx

```diff
@@ -231,62 +214,60 @@ export const BedCapacity = (props: BedCapacityProps) => {
         </div>
       ) : (
         <div className={className}>
-          <SelectFormField
-            name="bedType"
-            id="bed-type"
-            label="Bed Type"
-            required
-            value={state.form.bedType}
-            options={bedTypes.filter((type) => !type.disabled)}
-            optionLabel={(option) => option.text}
-            optionValue={(option) => option.id}
-            onChange={handleChange}
-            disabled={!!id}
-            error={state.errors.bedType}
-          />
-          <div className="flex flex-col gap-7 md:flex-row">
-            <TextFormField
-              className="w-full"
-              id="total-capacity"
-              name="totalCapacity"
-              label="Total Capacity"
-              required
-              type="number"
-              value={state.form.totalCapacity}
-              onChange={handleChange}
-              error={state.errors.totalCapacity}
-              min={1}
-            />
-            <TextFormField
-              className="w-full"
-              id="currently-occupied"
-              label="Currently Occupied"
-              required
-              name="currentOccupancy"
-              type="number"
-              value={state.form.currentOccupancy}
-              onChange={handleChange}
-              error={state.errors.currentOccupancy}
-              min={0}
-              max={state.form.totalCapacity}
-            />
-          </div>
-
-          <div className="cui-form-button-group mt-4">
-            <Cancel onClick={handleClose} />
-            {!isLastOptionType && headerText === "Add Bed Capacity" && (
-              <Submit
-                id="bed-capacity-save-and-exit"
-                onClick={(e) => handleSubmit(e, "Save and Exit")}
```

Comment:
Let's keep the two buttons (save and add more and save, instead of just save).

Reply:
I had used Form component in BedCapacity and StaffCapacity component.
This Form component is also used in PatientRegister component.
so any changes in Form component will also reflect in patient register page also.
That's the reason I had decided to remove "save and add more" button.

There is also one more way to add this button apart from adding it to Form component We can add it to individual component, but its affecting the UI

Reply:
Hmm, okay. We can add an additional button rendered conditionally in form component then (and pass in the button text from BedCapacity/StaffCapacity) because we do need it, especially for the facility creation page.

You can access that page at /facility/create (with superuser credentials, which you can create locally through the cli/[pgAdmin](https://www.pgadmin.org/)).

Reply:
For this I have got a solution.
I can add a prop named "showSaveAndAddMore" to show "Save and add more" button conditionally.